sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/executable/SonarTSCoreBundle.java

```diff
@@ -98,6 +97,14 @@ public SonarTSRunnerCommand getSonarTsRunnerCommand(String tsconfigPath, Iterabl
     return runnerCommand;
   }
 
+  /**
+   * Returns path to node.js executable
+   */
+  @Override
+  public String getNodeExecutable() {
+    return configuration.get(TypeScriptPlugin.NODE_EXECUTABLE).get();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=stas%2Fnode-version&issues=AWCicK81qDrCo5ysgJkn&open=AWCicK81qDrCo5ysgJkn)

[ISSUE_KEY]: AWCicK81qDrCo5ysgJkn